src/Native/Runtime/windows/CoffNativeCodeManager.cpp

```diff
@@ -296,10 +308,16 @@ void CoffNativeCodeManager::EnumGcRefs(MethodInfo *    pMethodInfo,
         codeOffset - 1 // TODO: Is this adjustment correct?
         );
 
+    ICodeManagerFlags flags = (ICodeManagerFlags)0;
+    if (pNativeMethodInfo->executionAborted)
+        flags = ICodeManagerFlags::ExecutionAborted;
+    if (IsFilter(pMethodInfo))
```

Comment:
Done, it seems the fix is exactly the same except for the mechanics of getting the unwind block flags.